cmd/create.go

```diff
@@ -61,9 +57,6 @@ func NewCommandCreate() *cobra.Command {
 
 	cmd.Flags().String("config", "", "loadManifest flags for this command from a file.")
 
-	cmd.Flags().Bool("dry-run", false, "Display what resources this corral will create.")
```

Comment:
LGTM, just curious as to why we removed this (I'm assuming it's because we can't guarantee valid terraform for this command with multiple modules)

Reply:
Yeah because the modules from before would not be setting the vars I don't see a way we  could reliably do this.